import_and_export_data/guides/create-connector.rst

```diff
@@ -313,3 +313,166 @@ If you don't see your changes, make sure you have run (``bin/console assets:inst
+
+.. code-block:: yaml
+    :linenos:
+
+    pimee_catalog_rule.job.xlsx_product_import_with_rules:
+        class: '%pim_connector.job.simple_job.class%'
+        arguments:
+            - '%pimee_catalog_rule.job_name.xlsx_product_import_with_rules%'
+            - '@event_dispatcher'
+            - '@akeneo_batch.job_repository'
+            -
+                - '@pim_connector.step.charset_validator'
+                - '@pimee_catalog_rule.step.xlsx_product.import'
+                - '@pimee_catalog_rule.step.xlsx_product.import_associations'
+                - '@pimee_catalog_rule.step.xlsx_product.execute_rules'
+            - true # <-- This property should be true if the job is stoppable
+        tags:
+            - { name: akeneo_batch.job, connector: '%pim_connector.connector_name.xlsx%', type: '%pim_connector.job.import_type%' }
+
+2. If your job uses tasklets (by implementing ``TaskletInterface``) to execute your business logic. You can inject the service `"akeneo_batch.job.job_stopper"` directly in your custom tasklets and use the function `JobStopper::isStopping` to know if user asked to stop the job. If this function return true, you should use the function `JobStopper::stop` and exit your Tasklet.
+
+The following example shows a simple tasklet able to stop when a user stops the job from the UI.
+
+.. code-block:: php
+    :linenos:
+
+    <?php
+    declare(strict_types=1);
+
+    namespace Acme\Bundle\StoppableJobBundle;
+
+    class TrackableTasklet implements TaskletInterface
+    {
+        private const BATCH_SIZE = 100;
+
+        protected ?StepExecution $stepExecution = null;
+        protected FindItemsToProcess $findItemsToProcess;
+        protected JobStopper $jobStopper;
+
+        public function __construct(
+            FindItemsToProcess $findItemsToProcess,
+            JobStopper $jobStopper
+        ) {
+            $this->jobStopper = $jobStopper;
+            $this->findItemsToProcess = $findItemsToProcess;
+        }
+
+        public function setStepExecution(StepExecution $stepExecution): void
+        {
+            $this->stepExecution = $stepExecution;
+        }
+
+        public function execute(): void
+        {
+            $itemsToProcess = $this->FindItemsToProcess->find();
+            foreach ($itemsToProcess as $i => $itemToProcess) {
+                $itemToProcess->doSomeWork();
+
+                // Check every 100 items if the process should be stopped
+                if ($i % self::BATCH_SIZE === 0
+                    && $this->jobStopper->isStopping($this->stepExecution)
+                ) {
+                    $this->jobStopper->stop($this->stepExecution);
+
+                    return;
+                }
+            }
+        }
+    }
+
+Track the progress of a job
+---------------------------
+
+As of now, it is possible your users are not able to see the progress of your job in the UI.
```

Comment:
If it is a 5.0 feature too, you need to say it explicitly I think (like you did to stop a job feature).